src/elements/Label/Readme.md

```diff
@@ -0,0 +1,26 @@
+### Usage
+```js static
+import { Label } from '@codespec/react-toolset'
+```
+
+### Basic Usage
+```js
+<Label>My Label</Label>
+```
+
+### Required Field Label
+```js
+<Label required={true}>
```

Comment:
```
<Label required>
``` 
is enough.

Reply:
You are right. That works. But why `required` is parsed as `{required: ture}` inside props?